Sprint-1/interpret/percentage-change.js

```diff
@@ -20,3 +20,17 @@ console.log(`The percentage change is ${percentageChange}`);
 // d) Identify all the lines that are variable declarations
 
 // e) Describe what the expression Number(carPrice.replaceAll(",","")) is doing - what is the purpose of this expression?
+
+
+// a) 1-carPrice = Number(carPrice.replaceAll(",", ""));   2-priceAfterOneYear = Number(priceAfterOneYear.replaceAll("," ""));
+//       3- console.log(`The percentage change is ${percentageChange}`);
```

Comment:
A function call is an expression and therefore can appear as a subexpression in another expression or as a parameter of 
     another function call. For example, there are 3 function calls happening in `console.log(Math.round(Number("12.34")))`.